app/controllers/errors_controller.rb

```diff
@@ -6,6 +6,11 @@ class ErrorsController < ApplicationController
   end
 
   def auth_error
-    @page_subtitle = "Auth Error"
+    @page_subtitle = t(".subtitle")
+  end
+
+  def timeout_error
+    @page_subtitle = t(".subtitle")
```

Comment:
This should use different locale keys, like "browser_title" to distinguish between this and the subtitle on the page itself